docs/product-portal/accounts/dev-process.md

```diff
@@ -40,178 +40,87 @@ for the Application Services team:
-We use labels and milestones to keep things organised,
-and [waffle.io](https://waffle.io) to provide
-an overview of bug status and activity:
-
-* [Active issues for Firefox Accounts](https://waffle.io/mozilla/fxa)
+We use labels and milestones to keep things organised.
 
 Issue status is reflected by the following:
 
-* The milestone indicates *why* we are working on this issue.
-* The waffle column indicates *what* the next action is and *when*
-  we expect to complete it.
+* The milestone indicates *when* we are working on this issue.
+* The issue itself will have updates indicating *what* the next action.
 * The assignee, if any, indicates *who* is responsible for that action.
 
 ### Milestones
 
-When we start working on a new feature,
+When we start working on a new sprint,
 we create a corresponding
 [milestone in github](https://github.com/mozilla/fxa/milestones)
-and break down the task
-into bugs associated with that milestone.
-There's also an ongoing
-["quality" milestone](https://waffle.io/mozilla/fxa?milestone=FxA-0:%20quality)
-for tracking work
-related to overall quality
-rather than a particular feature.
-
-If it's not obvious
-what milestone an issue should belong to,
-that's a strong signal
-that we're not ready to work on it yet.
-
-Milestones are synced across all our repos using the
-[sync_milestones.js](https://github.com/mozilla/fxa/blob/master/scripts/sync_milestones.js)
-script.
-
-### Waffle Columns
-
-Issues that are not being actively worked on are managed in the following columns:
-
-* **triage**:  all incoming issues start out in this column by default.
-* **backlog**: issues that we plan to work on someday, but not urgently.
-* **next**: issues that we plan to pick up in the next development cycle.
-
-Issues that are under active development are managed in the following columns:
+and fill it with issues that should be closed in that sprint.
+
+### Labels
+
+We use labels to help categorize and identify issues. The label taxonomy is:
+
+##### Bug Type/Status:
+* **status:unconfirmed**
+* **status:bug**
+* **status:enhancement**
+* **status:wontfix**
+* **status:invalid**
+* **status:duplicate**
+* **Status:worksforme**
+* **status:blocker**
+
+##### Categories:
+* **code quality**
+* **perf**
+* **security**
+* **documentation**
+
+##### Contributors:
+* **good first issue**
+* **help wanted**
+* **help wanted (taken)** Since you can't assign an issue to someone who isn't in the organization
+
+##### Skill labels should accompany any good first issue label:
+* **skill:css**
+* **skill:html**
+* **skill:js**
+* **skill:rust**
+* **skill:database**
+
+##### Blocked on another person/team/decision:
+* **needs:discussion** There is an open question blocking this issue from moving forward
+* **needs:legal**
+* **needs:ops**
+* **needs:product** Any product questions including missing content or copy
+* **needs:qa**
+* **needs:security**
+* **needs:visuals**  UI is in place but doesn’t look right
```

Comment:
visuals seems pretty close in concept to ux. I've never actually used it.

Reply:
It was a request from John to start differentiating.  UX = flows, Visuals = images/icons on the page in a nutshell (I'm sure there is other nuance).